cpp/test/sparse/dist_coo_spmv.cu

```diff
@@ -171,48 +202,56 @@ class SparseDistanceCOOSPMVTest
     dist_config.a_indptr = indptr;
     dist_config.a_indices = indices;
     dist_config.a_data = data;
-    dist_config.handle = cusparseHandle;
-    dist_config.allocator = alloc;
-    dist_config.stream = stream;
 
     int out_size = dist_config.a_nrows * dist_config.b_nrows;
 
-    allocate(out_dists, out_size);
+    allocate(out_dists_dense, out_size);
+    allocate(out_dists_hash, out_size);
+    allocate(out_dists_chunk, out_size);
 
     run_spmv();
 
-    CUDA_CHECK(cudaStreamSynchronize(stream));
+    CUDA_CHECK(cudaStreamSynchronize(handle.get_stream()));
   }
 
   void TearDown() override {
-    CUDA_CHECK(cudaStreamSynchronize(stream));
+    CUDA_CHECK(cudaStreamSynchronize(handle.get_stream()));
     CUDA_CHECK(cudaFree(indptr));
     CUDA_CHECK(cudaFree(indices));
     CUDA_CHECK(cudaFree(data));
-    CUDA_CHECK(cudaFree(out_dists));
+    CUDA_CHECK(cudaFree(out_dists_dense));
+    CUDA_CHECK(cudaFree(out_dists_hash));
+    CUDA_CHECK(cudaFree(out_dists_chunk));
     CUDA_CHECK(cudaFree(out_dists_ref));
   }
 
   void compare() {
     raft::print_device_vector("expected: ", out_dists_ref,
                               params.out_dists_ref_h.size(), std::cout);
-    raft::print_device_vector("out_dists: ", out_dists,
+    raft::print_device_vector("out_dists_chunk: ", out_dists_chunk,
```

Comment:
If we must keep the prints, we should probably only do them when the results don't match (I admit I'm guilty of keeping prints around in tests).